protocol/consensus.go

```diff
@@ -94,13 +94,18 @@ const ConsensusV17 = ConsensusVersion(
 	"https://github.com/algorandfoundation/specs/tree/5615adc36bad610c7f165fa2967f4ecfa75125f0",
 )
 
+// ConsensusV18 adds the ability to issue a transaction that marks an account non-participating
+const ConsensusV18 = ConsensusVersion(
+	"---->!!!TODO!!!<----",
```

Comment:
We'll want to PR a spec change into github.com/algorandfoundation/spec.
(Side note: It might or might not make sense to combine this protocol upgrade with Tsachi's protocol upgrade for fixing the reward rate calculation.)